star/private/diffusion_procs.f90

```diff
@@ -1184,7 +1184,7 @@ subroutine setup_struct_info( &
 
 
          subroutine do1(k)
-            use mod_typical_charge, only: eval_typical_charge
+            use ionization_lib, only: eval_typical_charge
```

Comment:
`eval_typical_charge` got ingested into the `star` module when we deleted ionization, so we'll need to make a decision about whether to get ride of `mod_typical_charge` from `star` and go back to the old way of doing things, or keep this new way.

The ionization module is mostly about something else, so I think my preference would be to keep `eval_typical_charge` as its own independent thing still living in `star`, and delete the things associated with it out of the `ionization` module.